airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -264,6 +264,66 @@ def fetch_job_metrics_by_id(self, job_id: str) -> dict:
         self.log.debug("fetch_job_metrics_by_id %s:\n%s", job_id, result)
         return result
 
+    def _fetch_list_job_messages_responses(self, job_id: str) -> Generator[dict, None, None]:
+        """
+        Helper method to fetch ListJobMessagesResponse with the specified Job ID.
+
+        :param job_id: Job ID to get.
+        :type job_id: str
+        :return: yields the ListJobMessagesResponse. See:
+            https://cloud.google.com/dataflow/docs/reference/rest/v1b3/ListJobMessagesResponse
+        :rtype: Generator[dict, None, None]
+        """
+        request = (
+            self._dataflow.projects()
+            .locations()
+            .jobs()
+            .messages()
+            .list(projectId=self._project_number, location=self._job_location, jobId=job_id)
+        )
```

Comment:
Hm. Why pylint is not complaining about `no-member` as it does in all other places where we use discovery endpoints?

Reply:
the variable `self.dataflow` is of the type Any.